app/views/YouNeedToRegisterView.scala.html

```diff
@@ -35,9 +35,11 @@
     @formHelper(action = YouNeedToRegisterController.onSubmit(), Symbol("autoComplete") -> "off") {
 
         @heading("youNeedToRegister.heading")
+        <p> @messages("youNeedToRegister.p1")</p>
```

Comment:
The font of the text does not match the rest of the service in English or Welsh
<img width="708" alt="image" src="https://github.com/user-attachments/assets/032e64f5-6ac7-424c-be77-74bf6614f1cd">

Also, on the translation document the english text does not contain a comma in the final line of text, the Welsh translation does, but when implementing it you have not put a comma in either language, is that an oversight? Or is it a change that was agreed? i.e. remove the comma from both.

Reply:
The comma needs to be added. UCD team confirms that we should align with the welsh document. I will make that update, Thanks for pointing it out!